content/docs/command-reference/dag.md

```diff
@@ -44,6 +44,8 @@ restore one or more pipelines later (see `dvc repro`).
   [DOT](<https://en.wikipedia.org/wiki/DOT_(graph_description_language)>)
   format. It can be passed to third party visualization utilities.
 
+- `-o`, `--outs` - show DAG of stage outputs instead of stages.
```

Comment:
```suggestion
- `-o`, `--outs` - show the graph of stage outputs instead of stage names.
```

Reply:
But I'm not 100% sure of this. Can `--outs` actually change the graph itself? Like, showing more nodes if a stage has multiple outputs?

Reply:
Yes, the graph changes completely
Without the options graph contains stages like
```
    +-------------------+  
    | data/data.xml.dvc |  
    +-------------------+  
              *            
              *            
              *            
         +---------+       
         | prepare |       
         +---------+       
              *            
              *            
              *            
        +-----------+      
        | featurize |      
        +-----------+      
         **        **      
       **            *     
      *               **   
+-------+               *  
| train |             **   
+-------+            *     
         **        **      
           **    **        
             *  *          
        +----------+       
        | evaluate |       
        +----------+       
```
but when `--outs` is added nodes are outputs (data, metrics, params)
```
                  +---------------+                    
                  | data/data.xml |                    
                  +---------------+                    
                           *                           
                           *                           
                           *                           
                  +---------------+                    
                  | data/prepared |                    
                  +---------------+                    
                           *                           
                           *                           
                           *                           
                  +---------------+                    
                  | data/features |                    
                **+---------------+****                
            ****           *           *****           
       *****               *                ****       
   ****                    *                    ****   
***                 +-----------+                   ***
  **                | model.pkl |                  **  
    **              +-----------+                **    
      **            **           **            **      
        **        **               **        **        
          **    **                   **    **          
      +-------------+             +----------+         
      | scores.json |             | prc.json |         
      +-------------+             +----------+         
```

Reply:
OK, thanks for confirming. So maybe something like
```suggestion
- `-o`, `--outs` - show a DAG of chained dependencies and outputs instead
  of the stages themselves. The graph may be significantly different.
```
> Prob. needs formatting

Reply:
BTW the DAG starts with a dependency (not necessarily an output) so can't the option name `outs` be misleading?

Reply:
p.s. that's a great example too! Mind adding it as an actual Example in the doc, @JIoJIaJIu ?